spec/features/components_guide_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.feature "Visit Component Guide" do
+  include ComponentGuideSteps
```

Comment:
It seems superfluous to have a helper file for this. I'd just add any methods into here (as per: https://github.com/alphagov/collections-publisher/blob/master/spec/features/access_control_spec.rb#L42-L62)

Reply:
Done